### ISSUE TITLE
Tomcat and Wildfly Links

### ISSUE THREAD

**Issue by christianwoehrle** [fabric8io/agent-bond#4]
Hi Roland,
the links above are broken. Are the samples still available somewhere?
Cheer Christian

**Comment by davsclaus** [comment]
Yes @rhuss the main readme has broken links in the bottom. Seems those samples are no longer there.

**Comment by gunnarmorling** [comment]
Ha @rhuss, just wanted to report those broken links. Seems I'm not the first one, though :)

**Comment by rhuss** [comment]
ok, ok, you got me.

For some reason, I'm not really aware of anymore ;-), I removed this directory. You can find it still [here](https://github.com/fabric8io/agent-bond/tree/537ec41050de682ac9bb57ea371901199227a17b/docker). 

I will re-check why I did this and eventually add it back (even checking whether it still works ;-)

Thanks for reporting and thanks for your patience ;-)
